### ISSUE TITLE
Docs reminder to unset environmental variables

### ISSUE THREAD

**Issue by mattidupre** [nlf/dlite#147]
A lot of people will be trying this having already installed the Docker suite. Might want to make note of the need to unset environmental variables in order for `docker` and `docker-compose` to work.

```
unset DOCKER_HOST
unset DOCKER_CERT_PATH
unset DOCKER_TLS_VERIFY
unset DOCKER_MACHINE_NAME
```

Then again, I'm new to Docker so maybe it's obvious. Either way, great project! Loving it.

**Comment by nlf** [comment]
good idea! feel free to submit a pull request. otherwise i'll get to it when i have a few minutes
